### ISSUE TITLE
BCP 47 alias

### ISSUE THREAD

**Issue by xfq** [specinfra/specref#688]
In https://www.specref.org/?q=bcp47 I see:

```
[BCP47]
Alias of [rfc5646].
```

But I think BCP 47 contains two RFCs, RFC 5646 and RFC 4647.

/cc @r12a

**Comment by marcoscaceres** [comment]
Yeah true... this is an interesting case. I wonder if BCP47 should be treated as it's own entry and point to https://www.rfc-editor.org/info/bcp47 ... I'm thinking that this is not an alias and maybe more things will become part of BCP47 over time.

@mnot, would you know how "bcp" things work at the IETF? Could more docs be added?

**Comment by mnot** [comment]
Yep, BCPs and STDs can refer to one or more RFCs, and they can change over time.

**Comment by marcoscaceres** [comment]
ok, then yeah... BCP should point to https://www.rfc-editor.org/info/bcp47 ... @xfq, do you want to have a go at updating the entry?

**Comment by marcoscaceres** [comment]
@tobie, alternatively, I wonder if the IETF bot script should be getting all the "BCP"s somehow?

**Comment by tobie** [comment]
> alternatively, I wonder if the IETF bot script should be getting all the "BCP"s somehow?

Yeah. I think that makes sense. Not sure where to get them from, though.

**Comment by aphillips** [comment]
@marcoscaceres @tobie Is this stalled on getting a proposal from one of us (@xfq or perhaps from me)? BCP47 is heavily referenced in the I18N space.

**Comment by tobie** [comment]
Yeah, proposal plus ideally pull request.

**Comment by aphillips** [comment]
How about just a PR? 😸 #962.
